qa/tasks/cephfs/test_volumes.py

```diff
@@ -7432,6 +7446,159 @@ def test_subvolume_under_group_snapshot_clone(self):
         # verify trash dir is clean
         self._wait_for_trash_empty()
 
+    def test_subvolume_snapshot_clone_with_no_wait_enabled(self):
+        subvolume = self._gen_subvol_name()
+        snapshot = self._gen_subvol_snap_name()
+        clone1, clone2, clone3 = self._gen_subvol_clone_name(3)
+
+        # create subvolume
+        self._fs_cmd("subvolume", "create", self.volname, subvolume, "--mode=777")
+
+        # do some IO
+        self._do_subvolume_io(subvolume, number_of_files=10)
+
+        # snapshot subvolume
+        self._fs_cmd("subvolume", "snapshot", "create", self.volname, subvolume, snapshot)
+
+        # Decrease number of cloner threads
+        self.config_set('mgr', 'mgr/volumes/max_concurrent_clones', 2)
+        max_concurrent_clones = int(self.config_get('mgr', 'mgr/volumes/max_concurrent_clones'))
+        self.assertEqual(max_concurrent_clones, 2)
+
+        # Enable the snapshot_clone_no_wait config option
+        self.config_set('mgr', 'mgr/volumes/snapshot_clone_no_wait', True)
+        threads_available = self.config_get('mgr', 'mgr/volumes/snapshot_clone_no_wait')
+        self.assertEqual(threads_available, 'true')
+
+        # Insert delay of 15 seconds at the beginning of the snapshot clone
+        self.config_set('mgr', 'mgr/volumes/snapshot_clone_delay', 15)
+
+        # schedule a clone1
+        self._fs_cmd("subvolume", "snapshot", "clone", self.volname, subvolume, snapshot, clone1)
+
+        # schedule a clone2
+        self._fs_cmd("subvolume", "snapshot", "clone", self.volname, subvolume, snapshot, clone2)
+
+        # schedule a clone3
+        cmd_ret = self.run_ceph_cmd(
```

Comment:
causes this failure:

https://pulpito.ceph.com/yuriw-2024-03-12_14:59:27-fs-wip-yuri11-testing-2024-03-11-0838-reef-distro-default-smithi/7593684/

Reply:
We are seeing this failure as https://github.com/ceph/ceph/pull/50569 isn't backported to reef yet. I have updated the PR by changing `self.run_ceph_cmd()` to `self.mgr_cluster.mon_manager.run_cluster_cmd()` .

Reply:
https://github.com/ceph/ceph/pull/54945 is the reef backport that brings in that helper.

Reply:
cc @rishabh-d-dave

Reply:
If that's the case, I can see yuri's testing label on that PR. I will have to revert this change back after it gets merged. Should I remove needs-qa label until that gets merged?

Reply:
@rishabh-d-dave Is https://github.com/ceph/ceph/pull/54945 good to merge?

Reply:
@neesingh-rh changed this too use run_cluster_cmd